internal/cli/alpha/manifestgen/templates/helm-implementation.yaml.tmpl

```
@@ -0,0 +1,154 @@
+ocfVersion: 0.0.1
+revision: {{ .Revision }}
+kind: Implementation
+metadata:
+  prefix: "cap.implementation.{{ .Prefix }}"
+  name: {{ .Name }}
+  displayName: "{{ .Name }} Action"
+  description: "{{ .Name }} Action"
+  documentationURL: https://example.com
+  supportURL: https://example.com
+  maintainers:
+    - email: dev@example.com
+      name: Example Dev
+      url: https://example.com
+  license:
+    name: "Apache 2.0"
+
+spec:
+  appVersion: "1.0.x" # TODO(ContentDeveloper): Set the supported application version here
+  additionalInput:
+    parameters:
+      typeRef:
+        path: "cap.type.{{ .Prefix }}.{{ .Name }}-input"
+        revision: 0.1.0
```

```suggestion
  additionalInput:
    parameters:
      additional-parameters:
        typeRef:
          path: "cap.type.{{ .Prefix }}.{{ .Name }}-input"
          revision: 0.1.0
```

after #450 syntax was changed.